site/content/en/docs/Installation/Install Agones/helm.md

```diff
@@ -233,6 +233,7 @@ The following tables lists the configurable parameters of the Agones chart and t
 | `agones.allocator.replicas`                              | The number of replicas to run in the deployment                                                                                                                                                                                                                                                                  | `3`                                       |
 | `agones.allocator.service.name`                          | Service name for the allocator                                                                                                                                                                                                                                                                                   | `agones-allocator`                        |
 | `agones.allocator.service.serviceType`                   | The [Service Type][service] of the HTTP Service                                                                                                                                                                                                                                                                  | `LoadBalancer`                            |
+| `agones.allocator.service.grpc.appProtocol`                 | If the ServiceType is set to "appProtocol",  this is useful to connect gRPC with GKE gateway                                                                                                                                                                                                    | `""`                             |
```

Comment:
We'll need to make a copy of the full table and do `feature` shortcodes like we normally do.

Maybe documentation should read:

> The `appProtocol` to set on the Service for the gRPC allocation port. If left blank, no value is set.

Reply:
The changes are present in the feature shortcodes and the description has been updated as per your suggestion.

Reply:
Manual testing for the new changes can be found here - https://gist.github.com/Kalaiselvi84/cebc2fc16a40d3c5faf8750e0e539429